packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -248,8 +250,10 @@ export abstract class InversifyHttpAdapter<
       return this._replyText(request, response, body);
     } else if (body === undefined || typeof body === 'object') {
       return this._replyJson(request, response, body);
-    } else {
+    } else if (typeof body === 'number' || typeof body === 'boolean') {
       return this._replyText(request, response, JSON.stringify(body));
+    } else {
+      return this._replyStream(request, response, body);
```

Comment:
This code is unreachable. A `Stream` `body` is an `object`.

Reply:
Fixed